### PR TITLE
refactor: logic for getting report attributes

### PR DESCRIPTION
**Pull request by anthonyamaro15**
Jira ticket: https://gfw.atlassian.net/browse/MAPB-130
test link: https://alpha.blueraster.io/gfw-mapbuilder/pull-requests/MAPB-130/?appid=0174a49d49244b1cb57f7491c173144c

@davidlstarr  once this is approved I will remove test web map before merging,

Test cases tested:
1. Drawing polygon
2. Selecting polygon

### CODE REVIEW

**Comment by davidlstarr** [comment]
@anthonyamaro15 I'm seeing just an outline for the polygon on the report page. Is that correct?

Screen Shot 2023-02-17 at 10 30 06 AM

File: configs/urls-config.ts [hunk 1]
```diff
@@ -0,0 +1,3 @@
+export const URLS = {
+  GEOSTORE: 'https://production-api.globalforestwatch.org/v1/geostore/',
```
Comment (davidlstarr) [inline configs/urls-config.ts]: Is this file really needed if its only used once in the code?

**Review by davidlstarr — CHANGES_REQUESTED** [review]

**Comment by anthonyamaro15** [comment]
@davidlstarr  for the outline polygon, we can create a new ticket for that since the issue is not related to the print issue.

**Comment by davidlstarr** [comment]
@anthonyamaro15 Is the client aware of this issue?

File: configs/urls-config.ts (hunk 1 above)
Comment (anthonyamaro15) [reply configs/urls-config.ts]: @davidlstarr I will be adding endpoints here as I go to isolate endpoints to a single file

**Comment by anthonyamaro15** [comment]
@davidlstarr  I don't think so, this only happens to some of the polygons

File: configs/urls-config.ts (hunk 1 above)
Comment (davidlstarr) [reply configs/urls-config.ts]: Are there any other endpoints in the app atm? I honestly don't think this is needed and we can create this when there are multiple endpoints.

**Comment by davidlstarr** [comment]
@anthonyamaro15 Go ahead and create a Jira ticket in the backlog for this item

**Review by davidlstarr — APPROVED** [review]
